README.md

```diff
@@ -1,2 +1,88 @@
 # mtrho-operator
-Migration Toolkit for Kubernetes Operator
+Migration Toolkit for Red Hat OpenShift operator.
+
+#### Prerequisite
+
+<b>Note</b>: 
+- mtRHO is compatible with OpenShift 4.10/4.10+ versions.
+- Go 1.18 version needed to build operator
+
+
+
+1. Install pipeline operator from Operator-Hub
+2. Make sure `CusterTask`, `Pipeline` CRDs are available after the installation of Pipeline operator. It might take a minute or two before the CRDs appear.
+
+#### Building custom images
```

Comment:
I think we should document:
1. How to install via our published index image (maybe this is a follow-up PR)
1. How to create a bundle + index + install via custom index
1. Developer, how to build an image and install the operator from manifests directly (thinking `make deploy` `make install`).